ivy/functional/backends/paddle/experimental/losses.py

```diff
@@ -0,0 +1,29 @@
+# global
+import math
+
+import paddle
+import paddle.nn.functional as F
+from ivy.utils.exceptions import IvyNotImplementedException
+from typing import Optional, Tuple
+
+import ivy
+
+from .. import backend_version
+
+from ivy.functional.ivy.experimental.linear_algebra import _check_valid_dimension_size
+
```

Comment:
add the `@with_unsupported_device_and_dtypes` handler as this function doesn't support all dtypes